website/static/css/custom.css

```diff
@@ -403,3 +403,23 @@ redoc .operation-type {
   background-color: #FEF0EB;
   border-radius: 15px;
 }
+
+/* Adding Custom CSS for Feedback section */
```

Comment:
Can you use the button color from openebs.io? Same size and colors as the "Copy" button at the bottom of the openebs.io page.